suseconnect-ng.spec

```diff
@@ -16,60 +16,44 @@
 #
 
 
-%global provider_prefix github.com/SUSE/connect-ng
-%global import_path     %{provider_prefix}
-
-# set this to enable hwinfo test in %check
-%bcond_with hwinfo
+%global project github.com/SUSE/connect-ng
 
 Name:           suseconnect-ng
-# the version will get set by the 'set_version' service
-Version:        1.1.0~git2.f42b4b2
+Version:        1.7.0
 Release:        0
 URL:            https://github.com/SUSE/connect-ng
 License:        LGPL-2.1-or-later
 Summary:        Utility to register a system with the SUSE Customer Center
 Group:          System/Management
-Source:         connect-ng-%{version}.tar.xz
-Source1:        %name-rpmlintrc
+Source:         suseconnect-ng-%{version}.tar.xz
+Source1:        %{name}-rpmlintrc
 
 # Build against latest golang in Tumbleweed and
 # go1.18-openssl on all other distributions
```

Comment:
Maybe there's a special macro for Tumbleweed/Factory that's not related to the version number? Then we would not need to change this on each release.

Reply:
```suggestion
# go1.21-openssl on all other distributions
```

Reply:
problem is the `-openssl` version. I ask around if there is also a option to automatically default to the latest one

Reply:
I checked. The `golang(API)` is just a label for the `Provides:` which is added by the `go-x.xx` packages. Both the non and the openssl version does provide the same flag. This means it will not necessarily select the openssl version then. So no special handling for this. we need it hardcoded for now!